docs/references/listener.rst

```diff
@@ -0,0 +1,41 @@
+.. _listener:
+
+Listener
+########
+
+PostgREST uses `LISTEN <https://www.postgresql.org/docs/current/sql-listen.html>`_ to reload its :ref:`Schema Cache <schema_reloading_notify>` and :ref:`Configuration <config_reloading_notify>` via `NOTIFY <https://www.postgresql.org/docs/current/sql-notify.html>`_.
+This is useful in environments where you can’t send SIGUSR1 or SIGUSR2 Unix Signals.
+Like on cloud managed containers or on Windows systems.
+
+.. code:: postgresql
+
+  NOTIFY pgrst, 'reload schema'; -- reload schema cache
+  NOTIFY pgrst, 'reload config'; -- reload config
+  NOTIFY pgrst;                  -- reload both
+
+By default, the LISTEN channel is enabled (:ref:`db-channel-enabled`) and named ``pgrst`` (:ref:`db-channel`).
+
+Listener on Read Replicas
+=========================
+
+
+The ``LISTEN`` and ``NOTIFY`` commands do not work on PostgreSQL read replicas.
+Thus, if you connect PostgREST to a read replica the Listener will fail to start.
+
+.. code:: console
+
+  12/Jun/2024:15:19:57 -0500: Failed listening for notifications on the "pgrst" channel... connection to server... failed: session is read-only
```

Comment:
Ah, damn. I just realized this error doesn't make sense here, it's caused because of our `target_session_attrs=read-write`.

The previous one with plain SQL was much better. Maybe add something like:

```postgresql
-- check if the instance is a replica
postgres=# select pg_is_in_recovery();
 pg_is_in_recovery 
-------------------
 t
 (1 row)

postgres=# LISTEN pgrst;
<ERROR message here>
```